### PR TITLE
Fix lighting issue

### PR DESCRIPTION
**Pull request by arjxn-py**
https://github.com/user-attachments/assets/606d56d9-dccd-434b-bb4f-cb53c97290e8

### CODE REVIEW

**Comment by trungleduc** [comment]
what happens if we keep using the `MeshPhongMaterial`?

**Comment by arjxn-py** [comment]
> what happens if we keep using the `MeshPhongMaterial`?

`metalness` & `roughness` properties doesn't seem to be there in `MeshPhongMaterial`

**Comment by martinRenou** [comment]
The standard material uses PBR so it's actually good we're switching to this!

**Comment by trungleduc** [comment]
> > what happens if we keep using the `MeshPhongMaterial`?
> 
> `metalness` & `roughness` properties doesn't seem to be there in `MeshPhongMaterial`

i know, but are they needed if we use zero decay? Could you post a screenshot of using `MeshPhongMaterial`?

**Comment by arjxn-py** [comment]
> > > what happens if we keep using the `MeshPhongMaterial`?
> > 
> > 
> > `metalness` & `roughness` properties doesn't seem to be there in `MeshPhongMaterial`
> 
> i know, but are they needed if we use zero decay? Could you post a screenshot of using `MeshPhongMaterial`?

on it

**Comment by arjxn-py** [comment]
> i know, but are they needed if we use zero decay? Could you post a screenshot of using `MeshPhongMaterial`?


https://github.com/user-attachments/assets/2d73d287-7a78-4022-85bd-3983ceb789c3

@trungleduc

**Comment by trungleduc** [comment]
Thanks, indeed it looks better with the `MeshStandardMaterial`

**Review by trungleduc — APPROVED** [review]

File: packages/base/src/3dview/mainview.tsx [hunk 1]
```diff
@@ -227,7 +227,8 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      this._cameraLight = new THREE.PointLight(0xffffff, 1);
+      this._cameraLight = new THREE.PointLight(0xffffff, 100);
+      this._cameraLight.decay = 5;
```
Comment (martinRenou) [inline packages/base/src/3dview/mainview.tsx:231]: I thought we would be using decay `0` with an intensity of `1`?

Comment (arjxn-py) [reply packages/base/src/3dview/mainview.tsx:231]: Oh Shoot 😬 , this is my bad I think i pushed the wrong changes as I was changing them rapidly. So Sorry

Comment (martinRenou) [reply packages/base/src/3dview/mainview.tsx:231]: No worries!

**Pull request merged**